llama_stack/templates/starter/starter.py

```diff
@@ -78,22 +78,25 @@ def _get_model_entries_for_provider(provider_type: str) -> list[ProviderModelEnt
     if provider_type == "ollama":
         return [
             ProviderModelEntry(
-                provider_model_id="${env.OLLAMA_INFERENCE_MODEL:=__disabled__}",
+                provider_model_id="${env.OLLAMA_INFERENCE_MODEL:=}",
                 model_type=ModelType.llm,
+                enabled=False,
             ),
             ProviderModelEntry(
-                provider_model_id="${env.OLLAMA_EMBEDDING_MODEL:=__disabled__}",
+                provider_model_id="${env.OLLAMA_EMBEDDING_MODEL:=}",
                 model_type=ModelType.embedding,
                 metadata={
                     "embedding_dimension": "${env.OLLAMA_EMBEDDING_DIMENSION:=384}",
                 },
+                enabled=False,
```

Comment:
how does this get enabled?

Reply:
I'm changing the template to include: `enabled: ${env.ENABLE_OLLAMA:=False}"